src/targets/node/fetch/client.ts

```diff
@@ -107,16 +107,15 @@ export const fetch: Client = {
       }
     }
     blank();
-    push(`let url = '${url}';`);
-    blank();
```

Comment:
Removed this blank line because short snippets here were uncomfortably long to read.